### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Vani177**
I've added the CONTRIBUTING.md file, hope it works well!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+     
+     gh repo clone AyushSarode/Social-media-buttons.github.io
+     
+
+
+3. Create a new branch 
+4. Start hacking on the new branch
+5. Commit and push to the new branch
+6. Make a pull request towards the original repository (AyushSarode/Social-media-buttons)
```
Comment (AyushSarode) [inline CONTRIBUTING.md:33]: link my repo here

Comment (Vani177) [reply CONTRIBUTING.md:33]: sure, at it!

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ If you're new to git and/or GitHub, we suggest you go through [the GitHub Guides
 3. Create a new branch 
 4. Start hacking on the new branch
 5. Commit and push to the new branch
-6. Make a pull request towards the original repository (AyushSarode/Social-media-buttons)
+6. Make a pull request towards the original repository (AyushSarode/Social-media-buttons.github.io)
```
Comment (AyushSarode) [inline CONTRIBUTING.md:33]: That's incorrect. Proper way - ([username/repo name](repo link))

**Review by AyushSarode — APPROVED** [review]

**Comment by AyushSarode** [comment]
Thank you @Vani177!

**Comment by Vani177** [comment]
> Thank you @Vani177!

You're welcome @AyushSarode
Glad to contribute!

**Pull request merged**